pypeit/spectrographs/keck_mosfire.py

```diff
@@ -125,6 +126,12 @@ def default_pypeit_par(cls):
         par['sensfunc']['IR']['telgridfile'] = 'TelFit_MaunaKea_3100_26100_R20000.fits'
         return par
 
+
+
+    def get_ql_master_dir(self, file):
```

Comment:
Add doc string.  And see my comment above about generalizing.

Reply:
I think this method should provide a list of `meta` to be used to generate the master dir.  And a generic method should take that list and proceed.

Reply:
Done, let's do that in the future.

Reply:
Add a TODO